src/test/scala/tech/cryptonomic/conseil/routes/PlatformDiscoveryTest.scala

```diff
@@ -227,5 +229,33 @@ class PlatformDiscoveryTest extends WordSpec with Matchers with ScalatestRouteTe
         status shouldEqual StatusCodes.NotFound
       }
     }
+
+    "expose endpoint for cache initialization" in {
+      // given
+      (tezosPlatformDiscoveryOperations.initAttributesCount _).when().returns(())
+
+      // when
+      Post("/v2/metadata/cache") ~> addHeader("apiKey", "hooman") ~> sut(Map.empty) ~> check {
+
+        // then
+        status shouldEqual StatusCodes.OK
```

Comment:
Is this enough? Should it check the mock for an initialization call? I'm not sure we have another way to test if something changed from something observable as an http-route...

Reply:
I think it's enough, it just checks if endpoint exists and responds to the correct request

Reply:
fine then